pwa-devdocs/scripts/create-reference-docs/templates/renderClassDocs.js

```diff
@@ -0,0 +1,77 @@
+const requiredProp = '<i class="material-icons green">check_box</i>';
+
+/**
+ * Function for overriding template props
+ */
+const overrideProps = (props, propsOverrides) => {
+    if (propsOverrides == undefined) {
+        return props;
+    }
+    let result = Object.assign({}, props);
+
+    let propKeys = Object.keys(props);
+
+    propKeys.forEach(key => {
+        if (propsOverrides[key]) {
+            result[key] = propsOverrides[key];
+        }
+    });
+
+    return result;
+};
+
+/**
+ * Function for rendering a prop entry
+ */
+const renderPropEntry = ({ key, required, defaultValue, propDescription }) => {
+    return `| \`${key}\` | ${required} | ${defaultValue} | ${propDescription}|\n`;
+};
+
+/**
+ * Function for generating the table for props
+ */
+const renderClassPropsTable = props => {
+    let result = `## Props\n\n `;
+
+    let propsKeys = Object.keys(props);
```

Comment:
Try to get in the habit of using `const` for variables whose values won't change after assignment.